### ISSUE TITLE
Muted midi tracks being exported

### ISSUE THREAD

**Issue by anytizer** [LMMS/lmms#6849]
When a track is muted, it is still being exported in the .mid file.
It causes the midi player play the undesired tracks as well.

I found a similar issue: #6643.

**Comment by musikBear** [comment]
Yes you are correct. It is even worse. Muted tracks is being processed all the time, and use the same CPU as if they were unmuted. Atm that is not possible to change. The very core needs to be changed. It would be 2.0

**Comment by Veratil** [comment]
> Muted tracks is being processed all the time, and use the same CPU as if they were unmuted.

This is intentional because if you unmute it while playing then it will immediately have sound.

**Comment by musikBear** [comment]
> > Muted tracks is being processed all the time, and use the same CPU as if they were unmuted.
> 
> This is intentional because if you unmute it while playing then it will immediately have sound.

Just to make sure we are on the same page, i am talking about the diode-option MUTE, not the VOL-dial.
Dizzy called it an inferiority compared to the pro-DAWs  afair, and one that needed to be removed
If you believe that it could be truly removed from CPU-processing, then _toggle sound on/off_ in a track _during playback_ would be as easy as <combo-click>-VOL and let that toggle between previous volume and 0. That VOL==0 would then still be processed, but any track really MUTEd (via diode) would be truly out of the CPU

**Comment by anytizer** [comment]
Yes, Let us consider this thread to demand a simple solution - by just skipping to export the tracks that are set to be muted.
(And not discussing about processing of the muted tracks during playtime). It could be a feature enhancement, but not necessarily a bug.

The tracks that are muted with the permanent mute button aside the "solo" should not appear on the final midi (.mid) output while using File > Export Midi... Reason is that the .mid file could be a final product when exported from LMMS.

When a user wants to share the notes to some other DAWs or midi player, they should consider placing that green rectangular light "on" for that track.

![image](https://github.com/LMMS/lmms/assets/5563341/dbba60fc-3a0e-40ec-a11e-20cf2ce98708)

If the project has above like setup; only the first track should be "exported".
